### PR TITLE
Feature/tutorial

### PR DESCRIPTION
**Pull request by pgdaniel**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -20,4 +21,8 @@ end
 
 group :staging, :production do
   gem "thin", "~> 1.5.0"
-end
\ No newline at end of file
+end
+
+group :development do
+  gem "rails-default-database", "~> 1.0.6"
```
Comment (patbenatar) [inline Gemfile:27]: Nice +1

File: app/assets/javascripts/application.coffee [hunk 2]
```diff
@@ -11,6 +11,7 @@
 #= require backbone.localStorage
 #= require handlebars
 #
+#= require marked
 #= require github_flavored_markdown
```
Comment (patbenatar) [inline app/assets/javascripts/application.coffee]: I think we may be able to remove `github_flavored_markdown` now that we've updated Crevasse which I think switches from that to `marked` as its compiler dep

File: app/assets/javascripts/application/views/notes_list.coffee [hunk 3]
```diff
@@ -98,7 +100,15 @@ class Views.NotesList extends Backbone.View
   clearCurrentNote: ->
     @$title.val("")
     @refreshListItems()
-
+    
+  createIntroNotes: =>
```
Comment (patbenatar) [inline app/assets/javascripts/application/views/notes_list.coffee]: How come fat arrow here?

Comment (pgdaniel) [inline app/assets/javascripts/application/views/notes_list.coffee]: @patbenatar oh ya I'll fix needed it below to access notes

File: app/assets/javascripts/application/views/notes_list.coffee [hunk 4]
```diff
@@ -98,7 +100,15 @@ class Views.NotesList extends Backbone.View
...
+    $.getJSON("intro.json", (data) =>
+      $.each(data, (index, item) =>
+        note = new Note(title: item.titleVal, body: item.noteVal)
+        @notes.create(note)
+      )
+    )
```
Comment (patbenatar) [inline app/assets/javascripts/application/views/notes_list.coffee]: CoffeeScript function calls that take a callback function as their last argument look way prettier if you omit the wrapping parenthesis. Consider this:

``` coffeescript
$.getJSON "intro.json", (data) =>
  $.each(data, (index, item) =>
    note = new Note(title: item.titleVal, body: item.noteVal)
    @notes.create(note)
```

That's my personal aesthetic opinion, though. What do you think?

Also I believe you can just pass the note attributes to the collection's `create` method and it'll build the model for you—shouldn't need to explicitly create a model and pass the model to the collection.

File: public/intro.json [hunk 5]
```diff
@@ -0,0 +1,6 @@
+[
+  {
+    "noteVal": "# Welcome to Notational Momentum!\n\n## Shortcuts\n\n### In the order of most to least useful:\n\n| Key Combo  | Description\n|------------|--------------------------------------------|\n| `⌃ L`      | Place insertion point in search/title area\n| `Return`   | Begin editing note, creating it if necessary\n| `⌃ Escape` | Delete the selected note)",
+    "titleVal": "Shortcuts"
+  }
```
Comment (patbenatar) [inline public/intro.json:5]: Why not use "body" and "title" as the keys here so the JSON naming matches up with the model attribute naming?

File: app/assets/javascripts/application/views/notes_list.coffee (hunk 4 above)
Comment (pgdaniel) [inline app/assets/javascripts/application/views/notes_list.coffee]: @patbenatar Nice!  wasn't aware you could do that... with the parens

Comment (patbenatar) [inline app/assets/javascripts/application/views/notes_list.coffee]: Yep parens around args are optional in CS

Comment (pgdaniel) [inline app/assets/javascripts/application/views/notes_list.coffee]: @patbenatar took them out and it broke...

Comment (patbenatar) [inline app/assets/javascripts/application/views/notes_list.coffee]: Did you remove the opening parens too? Push the broken code to another branch—I'll take a look. I'm certain it's doable

Comment (pgdaniel) [inline app/assets/javascripts/application/views/notes_list.coffee]: @patbenatar nevermind forgot a match!

**Comment by pgdaniel** [comment]
@patbenatar Think should be ready to go

**Comment by patbenatar** [comment]
Nice! Looks great. :shipit:

**Pull request merged**